### PR TITLE
All fixed and added findByProjectName function

### PR DESCRIPTION
**Pull request by DmitryBobryakov**

### CODE REVIEW

File: tms/src/main/java/com/netcracker/edu/tms/dao/ProjectDaoJpaImpl.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    @Override
+    public boolean deleteProject(BigInteger projectId) {
+        Project same = entityManager.find(Project.class, projectId);
+
+        if (same != null) {
+            entityManager.remove(same);
+            LOGGER.debug("deleteProject called with {}", projectId.toString());
+            return true;
+        } else {
+            LOGGER.debug("No such Project in deleteProject appeared");
```
Comment (ViktorSuharev) [inline tms/src/main/java/com/netcracker/edu/tms/dao/ProjectDaoJpaImpl.java]: to be similar to updateProject method let's use WARN logging level and specify in
LOGGER.warn("No such Project with ID= {} in deleteProject appeared", projectId);

File: tms/src/main/java/com/netcracker/edu/tms/dao/ProjectDaoJpaImpl.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            return false;
+        }
+    }
+
+    @Override
+    public List<Project> findProjectsByCreatorId(BigInteger creatorId) {
+        if (creatorId.compareTo(BigInteger.ZERO) <= 0) {
```
Comment (ViktorSuharev) [inline tms/src/main/java/com/netcracker/edu/tms/dao/ProjectDaoJpaImpl.java]: More common practice is avoiding NPE (if creatorId is null) in methods like this is swapping object and argument:
BigInteger.ZERO.compareTo(creatorId) >= 0
https://stackoverflow.com/questions/5712100/interview-java-equals

File: tms-ui/src/main/frontend/src/app/component/App.js [hunk 3]
```diff
@@ -1,22 +1,34 @@
 import React from "react";
 import HeaderPanel from "./header/HeaderPanel";
+import ArticleList from "./header/ArticleList";
+import articles from "./header/articles";
+import 'bootstrap/dist/css/bootstrap.css';
+import ProjectHeader from "./header/ProjectHeader";
```
Comment (ViktorSuharev) [inline tms-ui/src/main/frontend/src/app/component/App.js]: Imported ProjectHeader does not exist (Probably you forgot to commit it)

File: tms-ui/src/main/frontend/src/app/component/App.js [hunk 4]
```diff
@@ -1,22 +1,34 @@
 import React from "react";
 import HeaderPanel from "./header/HeaderPanel";
+import ArticleList from "./header/ArticleList";
```
Comment (ViktorSuharev) [inline tms-ui/src/main/frontend/src/app/component/App.js]: Imported ArticleList does not exist
Probably you forgot to remove it

File: tms-ui/src/main/frontend/src/app/component/App.js [hunk 5]
```diff
@@ -1,22 +1,34 @@
...
+import articles from "./header/articles";
```
Comment (ViktorSuharev) [inline tms-ui/src/main/frontend/src/app/component/App.js]: Imported articles does not exist
Probably you forgot to remove it

File: tms/src/main/java/com/netcracker/edu/tms/dao/ProjectDaoJpaImpl.java [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    private EntityManager entityManager;
+
+    @Override
+    public Project getProjectById(BigInteger projectId) {
+        LOGGER.debug("getProjectById called with {}", projectId);
+        return entityManager.find(Project.class, projectId);
+    }
+
+    @Override
+    public Project getProjectByName(String name) {
```
[the remark this answers is not in the document]
Comment (DmitryBobryakov) [reply tms/src/main/java/com/netcracker/edu/tms/dao/ProjectDaoJpaImpl.java]: I think project's name must be unique in db level

**Pull request closed without merging**